File: features/promo/gamification/src/main/java/com/tokopedia/gamification/data/entity/GamificationSumCoupon.java
package com.tokopedia.gamification.data.entity;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * @author okasurya on 8/30/18.
 */
public class GamificationSumCoupon {
    @SerializedName("sumCoupon")
    @Expose
    private int sumCoupon;
    @SerializedName("sumCouponStr")
    @Expose
    private String sumCouponStr;

    public int getSumCoupon() {
        return sumCoupon;
    }

    public void setSumCoupon(int sumCoupon) {
        this.sumCoupon = sumCoupon;
    }

    public String getSumCouponStr() {
        return sumCouponStr;
    }

    public void setSumCouponStr(String sumCouponStr) {
        this.sumCouponStr = sumCouponStr;
    }
}

File: features/promo/gamification/src/main/java/com/tokopedia/gamification/data/entity/GamificationSumCoupon.kt
package com.tokopedia.gamification.data.entity

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class GamificationSumCoupon(
    @SerializedName("sumCoupon")
    @Expose
    var sumCoupon:Int = 0,

    @SerializedName("sumCouponStr")
@Expose
var sumCouponStr: String = "",
)